composer.json

```diff
@@ -1,30 +1,31 @@
 {
-  "name": "vendor-name/example-package",
-  "type": "package",
-  "description": "An example package for Roots Acorn.",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Brandon Nifong",
-      "email": "brandon@tendency.me"
+    "name": ":vendor_slug/:package_slug",
```

Comment:
Can we change back to 2 space indents for JSON? Just keeps it uniform with other Roots `composer.json` files.